test/uk/gov/hmrc/helptosavefrontend/forms/BankDetailsValidationSpec.scala

```diff
@@ -177,16 +177,12 @@ class BankDetailsValidationSpec extends TestSupport {
       "allow inputs" which {
 
         "have a length within the configured limits" in {
-          testRollNumber(Some("ab1"))(Right(Some("ab1")))
           testRollNumber(Some("ab12"))(Right(Some("ab12")))
+          testRollNumber(Some("length18stringonly"))(Right(Some("length18stringonly")))
         }
 
-        "contains trailing and leading spaces" in {
-          testRollNumber(Some("  ab1   "))(Right(Some("ab1")))
-        }
-
-        "contains control characters" in {
-          testRollNumber(Some("ab1\n\r\t"))(Right(Some("ab1")))
+        "which contain only dots, hyphens and forward slashes" in {
```

Comment:
it'd be good to have a test which validates against special characters which are not `.`, `/` or `-`

Reply:
we have a test case against special chars 
```
        "containing characters not allowed as per regex" in {
          testRollNumber(Some("a_b$c%&^@!"))(Left(Set(ErrorMessages.rollNumberInvalid)))
        }```

Reply:
oh yeah, sorry didn't see that